pkg/healthcheck/healthcheck.go

```diff
@@ -1199,6 +1199,29 @@ func (hc *HealthChecker) allCategories() []Category {
 						return validateControlPlanePods(hc.controlPlanePods)
 					},
 				},
+				{
+					description:         "control plane proxies are healthy",
+					hintAnchor:          "l5d-api-cp-proxy-healthy",
+					retryDeadline:       hc.RetryDeadline,
+					surfaceErrorOnRetry: true,
+					fatal:               true,
+					check: func(ctx context.Context) error {
+						return hc.CheckProxyHealth(ctx, hc.ControlPlaneNamespace, hc.ControlPlaneNamespace)
+					},
+				},
+				{
+					description: "control plane proxies and cli versions match",
```

Comment:
Now that we've included data plane checks by default. Do we still need the checks that are characterized under `control-plane-version`? It seems as if we are running some checks twice now based on this output:
```
linkerd-api
-----------
√ control plane pods are ready
√ control plane proxies are healthy
‼ control plane proxies and cli versions match
    linkerd-controller-8544bdbb77-8g6ft running git-ff0042f6 but cli running dev-87f6dbc0-dennis
    see https://linkerd.io/checks/#l5d-api-cp-cli-version for hints

linkerd-version
---------------
√ can determine the latest version
‼ cli is up-to-date
    unsupported version channel: dev-87f6dbc0-dennis
    see https://linkerd.io/checks/#l5d-version-cli for hints

control-plane-version
---------------------
‼ control plane is up-to-date
    unsupported version channel: git-ff0042f6
    see https://linkerd.io/checks/#l5d-version-control for hints
‼ control plane and cli versions match
    control plane running git-ff0042f6 but cli running dev-87f6dbc0-dennis
    see https://linkerd.io/checks/#l5d-version-control for hints

Status check results are √
```

Reply:
The `control-plane-version` check actually uses the version from the config and does not depend on the proxy versions of the control-plane components. Hence there is a difference

but I do also think it's very hard to create a case where they both are different 🤔